### ISSUE TITLE
WMS with more than 256 characters in keywordlist cannot be added as datasource

### ISSUE THREAD

**Issue by kartoffelfee** [mapbender/mapbender#705]
A WMS with more than 256 characters in keywordlist cannot be added as datasource.
Errormessage in app_dev.php:
```
An exception occurred while executing 'INSERT INTO mb_core_keyword (id, value, discriminator) VALUES (?, ?, ?)' with params [295, "Regionalplan; Regionalpl\u00e4ne; Regionalplanung; Region; Regionsgrenze; Raumordnung; Biotopverbundachse; Bodenschatz; Bodensch\u00e4tze; Gr\u00fcnzug; Hochwasser; Hochwasserschutz; Landschaftliches Vorbehaltsgebiet; L\u00e4rmschutz; L\u00e4rmschutzbereich; Regionaler Gr\u00fcnzug; Rohstoff; Trassenfestlegung; Trenngr\u00fcn; Vorbehaltsgebiet; Vorranggebiet; Wasser; Wasserversorgung; Wanderungskorridor; Wind; Windenergie; Windenergienutzung; infoMapAccessService; inspireidentifiziert", "wmssourcekeyword"]:

SQLSTATE[22001]: String data, right truncated: 7 ERROR: value too long for type character varying(255)
```
tested with:
http://risby.bayern.de/RisGate/servlet/Regionalplanung?REQUEST=GetCapabilities&SERVICE=WMS&VERSION=1.1.1

Here's a capabilites-xml of the server with a reduced keywordlist which works:
http://wms2016.wheregroup.com/risby.bayern.de/Regionalplanung.xml

**Comment by DeEgge** [comment]
This was fixed with https://github.com/mapbender/mapbender/commit/484a8df1c610235da71469ddc69f5a3970e984d0#diff-92745d90b30e21f4a2635ba116c421b8, maybe you are on an older version.

**Comment by werrolf** [comment]
Fix wurde nach 3.0.5 zurückportiert.

**Comment by AxxL** [comment]
We have to reconsider this solution because of incompatibilities of the database.
